mat/dense_test.go

```diff
@@ -2093,3 +2093,14 @@ func denseMulTransSymBench(b *testing.B, size int, rho float64) {
 		wd = &n
 	}
 }
+
+func BenchmarkDenseSum1000(b *testing.B) { denseSumBench(b, 1000, 100) }
+
+func denseSumBench(b *testing.B, size, iters int) {
+	b.StopTimer()
+	a, _ := randDense(size, 1.0, rand.NormFloat64)
+	b.StartTimer()
+	for i := 0; i < iters; i++ {
```

Comment:
This is not correct. The form of a benchmark loop is `for i := 0; i < b.N; i++ {`